src/test/kotlin/blackjack/domain/PlayerTest.kt

```diff
@@ -6,68 +6,77 @@ import org.junit.jupiter.api.assertAll
 
 class PlayerTest {
     @Test
-    fun `Player는 이름과 손패, 상태를 보관한다`() {
-        val player = Player.from("이름")
+    fun `Player는 이름과 손패, 점수를 보관한다`() {
+        val player = Player(
+            name = PlayerName("이름"),
+            initialCards = PlayingCards.empty()
+        )
 
         assertAll(
-            { assertThat(player.name).isEqualTo("이름") },
-            { assertThat(player.hands.cards).isEmpty() },
-            { assertThat(player.state).isInstanceOf(PlayerState.Start::class.java) }
+            { assertThat(player.name.value).isEqualTo("이름") },
+            { assertThat(player.cardsOfHands).isEmpty() },
+            { assertThat(player.score).isEqualTo(Score.zero()) }
         )
     }
 
     @Test
     fun `receive를 통해 카드를 받아 손패에 추가할 수 있다`() {
-        val player = Player.from("이름")
+        val player = Player(
+            name = PlayerName("이름"),
+            initialCards = PlayingCards.empty()
+        )
         player.receive(
             PlayingCards.from(
                 listOf(
-                    PlayingCard.of(Suit.CLUBS, CardNumber.JACK),
-                    PlayingCard.of(Suit.CLUBS, CardNumber.ACE)
+                    PlayingCard(Suit.CLUBS, CardNumber.JACK),
+                    PlayingCard(Suit.CLUBS, CardNumber.ACE)
                 )
             )
         )
 
         assertAll(
-            { assertThat(player.hands.cards).hasSize(2) },
-            { assertThat(player.state).isInstanceOf(PlayerState.Blackjack::class.java) }
+            { assertThat(player.cardsOfHands).hasSize(2) }
         )
     }
 
     @Test
-    fun `finish를 통해 더 이상 카드를 받지 않겠다는 의사를 나타낼 수 있다`() {
-        val player = Player.from("이름").apply {
-            receive(
-                PlayingCards.from(
-                    listOf(
-                        PlayingCard.of(Suit.CLUBS, CardNumber.JACK),
-                        PlayingCard.of(Suit.CLUBS, CardNumber.KING)
-                    )
+    fun `stay를 통해 더 이상 카드를 받지 않겠다는 의사를 나타낼 수 있다`() {
+        val player = Player(
+            name = PlayerName("이름"),
+            initialCards = PlayingCards.from(
+                listOf(
+                    PlayingCard(Suit.CLUBS, CardNumber.JACK),
+                    PlayingCard(Suit.CLUBS, CardNumber.KING)
```

Comment:
`CardNumber` 를 enum 으로 잘 활용해 주셨는데요, 다음과 같이 sealed 클래스를 활용해 보는 방법도 있을것 같습니다. 😀 

```suggestion
                    Jack(Suit.CLUBS),
                    King(Suit.CLUBS)
```